### ISSUE TITLE
UI of login and signup form 

### ISSUE THREAD

**Issue by MaahiVaghela** [Sahil-Gupta584/BooksMall#16]
login and sign-up form are not following the theme of the website.

**Comment by Abhilash107** [comment]
I want to contribute to this. 
Please assign it to me under Hacktoberfest'24 @Sahil-Gupta584 .

**Comment by Abhilash107** [comment]
@Sahil-Gupta584 Thanks!

**Comment by MaahiVaghela** [comment]
I raised this issue first so you should assign this to me @Sahil-Gupta584

**Comment by Sahil-Gupta584** [comment]
@MaahiVaghela Yes issues is opened by you but you didn't told about you want want work on so I assigned Abhilash107 .
But it's OK there are still many issue on websit you can work on it.
